velero-csi-plugin/csi_restorer.go

```diff
@@ -17,64 +17,89 @@ limitations under the License.
 package main
 
 import (
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	snapshotv1beta1api "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
 	corev1api "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
 	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
-	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
-	"k8s.io/apimachinery/pkg/api/meta"
 )
 
 // CSIRestorer is a restore item action plugin for Velero
 type CSIRestorer struct {
 	log logrus.FieldLogger
 }
 
-// AppliesTo returns information about which resources this action should be invoked for.
-// A RestoreItemAction's Execute function will only be invoked on items that match the returned
-// selector. A zero-valued ResourceSelector matches all resources.g
+// AppliesTo returns information about which resources the CSIRestorer action should be invoked for.
+// CSIRestorer RestoreItemAction plugin's Execute function will only be invoked on items that match the returned
+// selector. A zero-valued ResourceSelector matches all resources.
 func (p *CSIRestorer) AppliesTo() (velero.ResourceSelector, error) {
 	return velero.ResourceSelector{
 		IncludedResources: []string{"persistentvolumeclaims"},
+		//TODO: add label selector volumeSnapshotLabel
 	}, nil
 }
 
 // Execute allows the RestorePlugin to perform arbitrary logic with the item being restored,
```

Comment:
I would drop "allows the RestorePlugin to perform arbitrary logic" bit and just document *this* plugin's behavior.